src/nvim/if_cscope.c

```diff
@@ -123,11 +123,12 @@ char_u *get_cscope_name(expand_T *xp, int idx)
      * the pathname of the cscope database as argument.  Only complete
      * with connection numbers. -1 can also be used to kill all
      * connections. */
+    size_t i;
```

Comment:
This `i` shadows the `i` on line 90. This will work, but may be a little confusing.